runtime/gc_vlhgc/CopyForwardScheme.cpp

```diff
@@ -231,7 +231,7 @@ MM_CopyForwardScheme::initialize(MM_EnvironmentVLHGC *env)
 			return false;
 		}
 	}
-	if(omrthread_monitor_init_with_name(&_scanCacheMonitor, 0, "MM_CopyForwardScheme::cache")) {
+	if (omrthread_monitor_init_with_name(&_scanCacheMonitor, 0, "MM_CopyForwardScheme::cache")) {
```

Comment:
`omrthread_monitor_init_with_name` returns `intptr_t`: this test should be a comparison with zero:
```c
	if (0 != omrthread_monitor_init_with_name(&_scanCacheMonitor, 0, "MM_CopyForwardScheme::cache")) {
```